logic/common/file_utils.py

```diff
@@ -0,0 +1,101 @@
+'''
+This file contains:
+ - Header & Ending functions, for indicating the time elapsed in a successful run
+ - Reader function for choose the root from TOML
+
+'''
+
+#==================================================#
+'''Import'''
+
+import time
+import os
+import toml
+
+
+
+#==================================================#
+'''Print Functions'''
+
+str_linebreak = "=================================================="
+def print_linebreak():
+	print(str_linebreak)
+
+def print_linebreak_thick():
+	print("")
+	print(str_linebreak)
+	print("")
+
+def _print(print_str):
+	'''Alternate print function'''
+	print(f"\t{print_str}")
+
+def _print_list(list_s):
+	'''Print all the contents of a list of string'''
+	for line in list_s:
+		print(line)
+
+
+
+#==================================================#
+'''Header & End'''
+
+def start_procedures():
+	'''Template starting function'''
+	global time_beg
+	time_beg = time.time()
```

Comment:
I like beg because it has exactly the same amount of characters as "end". Kinda the OCD acting up on my end when I want everything to look "aligned" haha

```
time_beg = x
time_end = y
```

versus

```
time_start = x
time_end = y
```

EVERYTHING IS BETTER WHEN ALIGNED!!!